File: programme/check-leap-year/check-leap-year.cs
// Write a programme to check leap year
using System;

public class CheckLeapYear
{
    public static void Main(string[] args)
    {
        Console.Write("Input  : ");
        if (int.TryParse(Console.ReadLine(), out int num))
        {
            Console
                .WriteLine("Output :{0} Leap Year",
                DateTime.IsLeapYear(num) ? "" : " Not a");
        }
        else
        {
            Console.WriteLine("Input Err");
        }
    }
}

File: programme/check-leap-year/check-leap-year.jl
# Write a programme to check leap year

print("Input  : ")

year = parse(Float32, readline())

if year % 100 == 0
    if year % 400 == 0
        print("Output : Leap Year")
    else
        print("Output : Not a Leap Year")

    end

else
    if year % 4 == 0
        print("Output : Leap Year")

    else
        print("Output : Not a Leap Year")

    end
end
